FunctionalTableData/FunctionalTableData.h

```diff
@@ -27,4 +27,13 @@ static inline void catchAndRethrowException(__attribute__((noescape)) void (^ __
 	}
 }
 
+__attribute__((visibility("hidden")))
+static inline void catchException(__attribute__((noescape)) void (^ __nonnull inBlock)(void), __attribute__((noescape)) void (^ __nonnull exceptionHandler)(NSException *)) {
```

Comment:
Do we need this?

Reply:
Just for testing purposes, I could move that over to the test suite too if you'd like - instead of rethrowing, we'd just handle it (ex. fail or assert otherwise). Thought it was generic enough to stay in the main codebase.